19/transaction_network_election_v2.0.yaml

```diff
@@ -0,0 +1,53 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+type: object
+title: Network Election Schema - Propose a network change
+required:
+- operation
+- version
+- asset
+- outputs
+properties:
+  operation: "VALIDATOR_ELECTION"
```

Comment:
I like the idea of specific naming since we do not have a clear picture about other election use-cases yet. Can you also update the title and the file name?